h/views/groups.py

```diff
@@ -114,14 +114,15 @@ def _update_group(self, appstruct):
              request_method='GET',
              renderer='h:templates/groups/share.html.jinja2',
              effective_principals=security.Authenticated,
+             has_feature_flag=not_('search_page'),
              has_permission='read')
```

Comment:
Just want to highlight something from the commit message here - `has_feature_flag` is not strictly necessary here, without it Pyramid will still find `GroupSearchController` when the feature flag is _on_, but I like to be clear and explicit. See commit message.